app/code/Magento/Search/Test/Unit/Model/SynonymAnalyzerTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Search\Test\Unit\Model;
+
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+
+/**
+ * Class SynonymAnalyzerTest
+ */
+class SynonymAnalyzerTest extends \PHPUnit\Framework\TestCase
```

Comment:
Not sure this namespace exists within a scope of 2.1. Try to use `\PHPUnit_Framework_TestCase` instead

Reply:
Fixed and checked on 2.1 version.